polkadot/runtime/parachains/src/configuration/benchmarking.rs

```diff
@@ -13,45 +13,99 @@
 
 // You should have received a copy of the GNU General Public License
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+#![cfg(feature = "runtime-benchmarks")]
 
 use crate::configuration::*;
-use frame_benchmarking::{benchmarks, BenchmarkError, BenchmarkResult};
+use frame_benchmarking::v2::*;
 use frame_system::RawOrigin;
 use polkadot_primitives::{ExecutorParam, ExecutorParams, PvfExecKind, PvfPrepKind};
 use sp_runtime::traits::One;
 
-benchmarks! {
-	set_config_with_block_number {}: set_code_retention_period(RawOrigin::Root, One::one())
+#[benchmarks]
+mod benchmarks {
+	use super::*;
 
-	set_config_with_u32 {}: set_max_code_size(RawOrigin::Root, 100)
+	#[benchmark]
+	fn set_config_with_block_number() -> Result<(), BenchmarkError> {
+		#[extrinsic_call]
+		set_code_retention_period(RawOrigin::Root, One::one());
```

Comment:
This function (as well as `set_max_code_size` below) returns a `Result`, so that may be an Error. If you want the benchmarking function to return `Result<(), BenchmarkError>`, you should propagate the error from here. Otherwise, if the benchmarking function has an empty return type, you can `assert_ok!(...)` on the output of this function. Same below

Reply:
Yeah i see that they return a DispatchResult. How would one go to propagate the error from inside a extrinsic call?
Since those benchmarks previously didnt verify anything after the extrinsic call maybe the best call would be to just remove the return value ?

Reply:
Yeah, maybe it's better to remove the return value at this point since it's not really used